architecture/additional_concepts/authorization.adoc

```diff
@@ -351,3 +374,67 @@ on the request.
 
 If a matching set of constraints is found, then the pod is accepted. If the
 request cannot be matched to an SCC, the pod is rejected.
+
+==== SCC Prioritization
+
+SCCs have a priority field that affects the ordering when attempting to
+validate a request by the admission controller.  A higher priority
+SCC is moved to the front of the set when sorting.  When the complete set
+of available SCCs are determined they are ordered by:
+
+. Highest priority first, nil is considered a 0 priority
+. If priorities are equal, the SCCs will be sorted from most restrictive to least restrictive
+. If both priorities and restrictions are equal the SCCs will be sorted by name
+
+By default, the anyuid SCC granted to cluster administrators is given priority
+in their SCC set.  This allows cluster administrators to run pods as any
+user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
+administrator may still specify a `RunAsUser` if they wish.
+
+==== Understanding pre-allocated values and Security Context Constraints
+
+The admission controller is aware of certain conditions in the security context constraints that trigger
+it to look up pre-allocated values from a namespace and populate the security context constraint
+before processing the pod.
+
+The following security context constraints prompt the admission controller to look for pre-allocated
+values:
+
+. A RunAsUser strategy of MustRunAsRange with no min/max set.  Admission will look for the `openshift.io/sa.scc.uid-range`
+annotation to populate range fields.
+. An SELinux strategy of MustRunAs with no level set.  Admission will look for the `openshift.io/sa.scc.mcs`
+annotation to populate the level.
+. A FSGroup strategy of MustRunAsRange with no ranges set.  Admission will look for the
+`openshift.io/sa.scc.supplemental-groups` annotation.
+. A SupplementalGroups strategy of MustRunAsRange with no ranges set.  Admission will look for
+the `openshift.io/sa.scc.supplemental-groups` annotation.
+During the generation phase, the security context provider will default any values that are not
+specifically set by the user.  Defaulting is based on the strategy being used:
+
+. RunAsAny and MustRunAsNonRoot strategies do not provide default values
+. MustRunAs (single value) strategies provide the value that must be used
+. MustRunAsRange and MustRunAs (range based) strategies provide the min value of the range.
+If a range based strategy is configurable with multiple ranges it will provide the min value of
+the first configured range.
```

Comment:
If the above 3 points were meant to be nested, you can remove the white space on line 413 and add an extra `.` to the front of the 3 sub-points.